packages/amplify-util-mock/src/api/api.ts

```diff
@@ -92,11 +93,13 @@ export class APITest {
       const errMessage = 'Failed to start API Mocking.';
       context.print.error(errMessage + ' Running cleanup tasks.');
       await this.stop(context);
-      throw new AmplifyFault('MockProcessFault', {
-        message: `${errMessage}. Reason: ${e?.message}`,
-        link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url,
-      });
-    }
+      if (e.resolution == undefined || e.link == undefined ) {
+        context.print.red(`Reason: ${e.message}`);
+      } else {
+        context.print.red(`Reason: ${e.message}\nResolution: ${e?.resolution}`);
```

Comment:
Is there a scenario that `e` will be undefined here?

Reply:
yes, we don't necessarily have `Resolution:` & `Link:` to every Amplify Exception that is caught though `Reason:` is printed out which is the Error Message. So, for example in the PR description for scenario 3) adding blank schema in the schema.graphql file and mocking API 
we only have the error message which is printed out as `Reason:` & here the `Resolution:` & `Link:` are undefined. For reference:

https://github.com/aws-amplify/amplify-cli/pull/12537#:~:text=adding%20blank%20schema%20in%20the%20schema.graphql%20file%20and%20mocking%20API

<img width="512" alt="Screenshot 2023-04-24 at 11 21 36 PM" src="https://user-images.githubusercontent.com/107574718/235018052-30ce960a-6c0e-4cb4-ab3d-4f4bbf5fd540.png">

Reply:
I mean the code `${e?.resolution}` in the last line. If the error object itself is always defined, the `?` mark can be removed.